modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -31,6 +31,26 @@ The inference endpoint for the model is shown in the *Inference endpoint* field.
 // * `:443/api/v1/task/text-classification`
 // * `:443/api/v1/task/token-classification`
 
+*Caikit Standalone ServingRuntime*
+
+By defalt, the Caikit Standalone ServingRuntime exposes REST endpoints for use. You can also manually expose gRPC endpoints to suit your use case.
+
```

Comment:
also reword "You can also manually expose gRPC endpoints to suit your use case." to "If you want to use gRPC protocol instead, you will have to manually deploy a custom Caikit Standalone ServingRuntime."